plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/names/NameMatcher.kt

```diff
@@ -26,7 +26,8 @@ internal class NameMatcher(val name: MangledName) {
     }
 
     inline fun ifClassName(vararg segments: String, action: NameMatcher.() -> Unit) {
-        if (className == ClassKotlinName(segments.toList()))
+        // TODO: not sure that it is correct to add the second check here
+        if (className == ClassKotlinName(segments.toList()) || scopedName?.name == ClassKotlinName(segments.toList()))
```

Comment:
Ah, the reason for two different behaviours here is because `ifClassName` was checking for whether something is a member of a class, while this is checking whether it *is* this class.  Would using different functions for these purposes work?